### PR TITLE
teal module clinical blog post #123

### PR DESCRIPTION
**Pull request by khatril**

### CODE REVIEW

**Comment by bms63** [comment]
@pharmaverse/commswg  Hi all - does anyone have bandwidth to start looking over this blog post?

**Comment by StefanThoma** [comment]
> @pharmaverse/commswg Hi all - does anyone have bandwidth to start looking over this blog post?

Yes, I can take a look!

File: posts/2024-02-27_tmc_cran/tmc_cran.qmd [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+<!--------------- post begins here ----------------->
+
+![](teal.png){width="184"}
+
+## Introduction
+
+I am thrilled to share the exciting news of the release of `teal.modules.clinical` 0.9.0 on CRAN.
+This significant achievement marks a major milestone for the NEST team in our open-source efforts, that will make a profound impact on the entire open-source community.
```
Comment (StefanThoma) [inline posts/2024-02-27_tmc_cran/tmc_cran.qmd:28]:
```suggestion
This significant achievement marks a major milestone for the NEST team in our open-source efforts that will make a profound impact on the entire open-source community.
```

File: posts/2024-02-27_tmc_cran/tmc_cran.qmd [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+These resources showcase a huge range of examples of interactive visualizations using modules from this package, which can be reused and inspire you when building your teal shiny app.
+
+## The {teal} family
+
+This package release now completes our suite of `teal` packages that have been recently released to CRAN (you can read our other blog post [here!](https://pharmaverse.github.io/blog/posts/2024-02-13_teal_on_cran/teal_on_cran.html)).
+We invite you to delve deeper into the {teal} family of packages, including `teal.modules.clinical` by visiting our [teal website](https://insightsengineering.github.io/teal/latest-tag/).
+
+## Acknowledgments
+
+We would like to give a huge thanks to the hard work and dedication of the many developers (past and present) for making this release possible.
```
Comment (StefanThoma) [inline posts/2024-02-27_tmc_cran/tmc_cran.qmd:53]: Can you link to the github contributers here?

File: posts/2024-02-27_tmc_cran/tmc_cran.qmd [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+![](teal.png){width="184"}
+
+## Introduction
+
+I am thrilled to share the exciting news of the release of `teal.modules.clinical` 0.9.0 on CRAN.
+This significant achievement marks a major milestone for the NEST team in our open-source efforts, that will make a profound impact on the entire open-source community.
+
+## Accelerating clinical insights
+
+Designed to enable faster insights generation, the `teal.modules.clinical` package contains a set of standard teal modules to be used with CDISC data in order to generate many of the common analysis displays used in clinical trials.
```
Comment (StefanThoma) [inline posts/2024-02-27_tmc_cran/tmc_cran.qmd:32]: Could you always link to the package when mentioning it?
e.g: 
[{falcon}](https://pharmaverse.github.io/falcon/)

File: posts/2024-02-27_tmc_cran/tmc_cran.qmd [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+long_slug <- "2024-02-27_tmc_cran"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+![](teal.png){width="184"}
+
+## Introduction
+
```
Comment (StefanThoma) [inline posts/2024-02-27_tmc_cran/tmc_cran.qmd:26]: Could you briefly explain what teal is, in one sentence or so?

**Review by StefanThoma — CHANGES_REQUESTED** [review]
Great, succinct post. 
Just minor adjustments :)

**Comment by khatril** [comment]
Thanks @manciniedoardo @StefanThoma

**Comment by bms63** [comment]
@manciniedoardo feel free to publish and announce on slack/Linkedin!!

**Pull request merged**